lib/board.rb

```diff
@@ -66,10 +66,53 @@ def tie?
   end
 
   # return formatted board for printing to terminal
-  def display_board 
-    "#{@header}\n#{row_1}\n#{row_2}\n#{row_3}\n#{row_4}\n#{row_5}\n#{row_6}"
+  def pretty_print
+    array = ["ABCDEFG\n"]
+    (1..6).to_a.each do |number|
```

Comment:
this is much cleaner looking, glad we went over this, and it was easier than expected to refactor using what we already had

Reply:
Thanks again for your help as we worked through this, great team vibes.